managed-ledger/pom.xml

```diff
@@ -111,6 +111,12 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+      <dependency>
```

Comment:
Is there a way to avoid make managed-ledger depend on pulsar-broker-common ?

Reply:
No better way found

Reply:
Have you checkout the implementation of `org.apache.pulsar.broker.service.BrokerService#registerConfigurationListener` ? Can this listener work for the problem?

Reply:
I will fixed

Reply:
Fixed